cmd/ocitools/generate.go

```diff
@@ -382,3 +406,15 @@ func setupLinuxNamespaces(g generate.Generator, needsNewUser bool, nsMaps map[st
 		g.AddOrReplaceLinuxNamespace(nsName, nsPath)
 	}
 }
+
+func onlyExportFlagSpecified(context *cli.Context) bool {
+	onlyExportFlags := []bool{
+		context.Bool("seccomp-only"),
+	}
+	for _, flag := range onlyExportFlags {
+		if flag {
+			return true
```

Comment:
On Thu, Jul 21, 2016 at 01:55:45PM -0700, Grant Seltzer Richman wrote:
> Not disagreeing, but why? 

For one, it means that the implementation would respect --output
without needing special code in lots of places ;).  More broadly, if
this functionality is useful enough for a command-line option, it
seems like it deserves a place in the library API.  Passing Context
through SaveToFile to Save is a classic use of Context for tunneling
parameters [1], and makes it easy for library callers to say “Please
save just the seccomp stuff to this path” without having to roll their
own duplicate of [2] or pulling in all the other things setupSpec is
looking at.

[1]: https://blog.golang.org/context#TOC_5.
     “[A] Context parameter as the first argument to every function on
     the call path… ensures that critical values like security
     credentials transit Go programs properly.”
[2]: https://github.com/opencontainers/ocitools/pull/128/files#diff-3e0e09fa01dec94676136d9c0cf52025R374